workspaces/global-floating-action-button/plugins/global-floating-action-button/src/components/FloatingButton.tsx

```diff
@@ -52,6 +52,22 @@ export const FloatingButton = ({
 }) => {
   const { pathname } = useLocation();
   const fabButton = useStyles();
+  const [targetElement, setTargetElement] = React.useState<Element | null>(
+    null,
+  );
+
+  React.useEffect(() => {
+    const checkTargetElement = () => {
+      const element = document.querySelector('[class^="BackstagePage-root"]');
+      if (element) {
+        setTargetElement(element);
+      } else {
+        setTimeout(checkTargetElement);
```

Comment:
Isn't this resulting in an endless loop if the class isn't there? I think that is too dangerous.

Is there a reason why the label isn't available first, and then its available later?

Reply:
I have added delay to the setTimeout
PTAL